README.md

```diff
@@ -56,13 +58,20 @@ npm i --prefix src/frontend/
 
 # :penguin: Running the iNatAtor Application
 
-## Run backend in first terminal:
+## Run **database** in first terminal:
+1. Navigate to project root
+2. Launch **postgres container**:
+```bash
+docker compose up --build db
+```
+
+## Run **backend** in second terminal:
 1. Navigate to the main `ds4cg2024-inaturalist` directory if you are not already there:
 2. Launch the **backend**:
 ```bash
- uvicorn src.backend.app.main:app --reload
+ uvicorn src.backend.app.main:app --reload --env-file .env
```

Comment:
To run uvicorn and npm, we first need to set up the Conda environment and run npm I, why are these steps after? they should be higher up!